.github/workflows/lint-bpf-checks.yaml

```
@@ -101,11 +101,26 @@ jobs:
         with:
           persist-credentials: false
           fetch-depth: 0
+      - name: Set up Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: '1.24' 
```

This should follow the pattern used in other workflows, specifically:

```suggestion
      - name: Install Go                                                                                                                                                                                                        
        uses: actions/setup-go@d35c59abb061a4a6fb18e82ac0862c26744d6ab5 # v5.5.0                                                                                                                                                
        with:                                                                                                                                                                                                                   
          # renovate: datasource=golang-version depName=go                                                                                                                                                                      
          go-version: 1.24.3
```